src/webviews/src/StepConfig.tsx

```diff
@@ -33,41 +31,73 @@ export function StepConfig(props: StepConfigProps) {
 
   if ('with' in props.step && 'http_url' in props.step.with) {
     return (
-      <div>
-        <Input
-          value={props.step.with.downloaded_filename || ''}
-          placeholder="data.json"
-          title="Downloaded filename (required)"
-          label="The filename where you want the results to be saved. This file doesn't need to exist yet."
-          handleChange={e =>
-            handleHttpValueChange(
-              'downloaded_filename',
-              e.target.value || undefined
-            )
-          }
-        />
-        <Input
-          value={props.step.with.http_url}
-          title="Endpoint url (required)"
-          label="Which endpoint should we pull data from? This needs to be a stable, unchanging URL."
-          handleChange={e => handleHttpValueChange('http_url', e.target.value)}
-        >
-          <HttpEndpointPreview url={props.step.with.http_url} />
-        </Input>
-      </div>
+      <>
+        <div>
+          <VSCodeTextField
+            value={props.step.with.downloaded_filename || ''}
+            placeholder="data.json"
+            required
+            className="w-full"
+            onInput={
+              // @ts-ignore
+              e => {
+                handleHttpValueChange(
+                  'downloaded_filename',
+                  e.target.value || undefined
+                )
+              }
+            }
+          >
+            Downloaded Filename (required)
+          </VSCodeTextField>
+          <p className="text-[12px] mt-1 mb-0 text-[color:var(--vscode-descriptionForeground)]">
+            The filename where you want the results to be saved. This file
+            doesn't need to exist yet.
+          </p>
+        </div>
+        <div>
+          <VSCodeTextField
+            className="w-full"
+            onInput={
+              // @ts-ignore
+              e => {
+                handleHttpValueChange('http_url', e.target.value)
+              }
+            }
+            value={props.step.with.http_url}
+          >
+            Endpoint url (required)
```

Comment:
Forgot to add `required` attribute? That or should update the text field label?

Reply:
Good catch. I noticed that the presence of the `required` attribute doesn't affect the visual appearance of the label (e.g., a red asterisk). Is this expected?